__tests__/api.test.js

```diff
@@ -68,6 +68,16 @@ describe("Error Handlers", () => {
         );
       });
   });
+  test("GET commentsByID - 404 status with custom error message, when no comments for entered id", () => {
+    return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe(
+          "no comments found for the input id, sorry! Try a different review_id"
+        );
+      });
```

Comment:
this test should be a 200 as technically it's not a bad request, the review just has no comments - think how we could check if the review exists but has no comments